native/storybook/storybook.js

```diff
@@ -1,16 +1,14 @@
-/* eslint-disable global-require */
-
 import { AppRegistry } from 'react-native';
 import { getStorybookUI, configure } from '@storybook/react-native';
 
-// import stories
+/* eslint-disable global-require */
 configure(() => {
-  // require('./stories');
   require('../packages/react-native-bpk-component-text/stories');
 }, module);
+/* eslint-enable global-require */
+
+const StorybookUI = getStorybookUI({ onDeviceUI: false });
```

Comment:
Why aren't we specifying a port anymore?

Reply:
It's specified as a cli arg too.